ci-operator/step-registry/upi/gcp/rhel/pre/upi-gcp-rhel-pre-commands.sh

```diff
@@ -38,11 +38,23 @@ gcloud compute firewall-rules create "${INSTANCE_PREFIX}" \
   --network "${INSTANCE_PREFIX}" \
   --allow tcp:22,icmp
 
+# Steps should specify either an exact image by name, or an image family. Prioritize
+#  the exact image, then the image family, and fail otherwise. Defaults should be defined
+#  in the accompanying step ref.  IMAGE_OPTION expands to the full CLI flag and arg.
+if [ -n "$GOOGLE_COMPUTE_IMAGE_NAME" ]; then
+    IMAGE_OPTION="--image=$GOOGLE_COMPUTE_IMAGE_NAME"
+elif [ -n "$GOOGLE_COMPUTE_IMAGE_FAMILY" ]; then
+    IMAGE_OPTION="--image-family=$GOOGLE_COMPUTE_IMAGE_FAMILY}"
+else
+    >&2 echo 'GOOGLE_COMPUTE_IMAGE_NAME or GOOGLE_COMPUTE_IMAGE_FAMILY variables must be specified.'
+    exit 1
+fi
+
 gcloud compute instances create "${INSTANCE_PREFIX}" \
-  --image=rhel-8-v20220719 \
-  --image-project=rhel-cloud \
-  --zone "${GOOGLE_COMPUTE_ZONE}" \
-  --machine-type e2-standard-8 \
+  "$IMAGE_OPTION" \
```

Comment:
```suggestion
  $IMAGE_OPTION \
```
otherwise we get (note `'` wrapping --image-family):
```
gcloud compute instances create ci-op-s2pt6lyg-1e4b3 '--image-family=rhel-8}' --image-project=rhel-cloud 
```